docs/_includes/star-rating-select.html

```diff
@@ -0,0 +1,110 @@
+<div id="star-rating-select">
+    {% include section-header.html name="star-rating-select" version=page.versions.star-rating-select %}
+
+    <p>Star rating select allows users to interact with and set a star rating.</p>
+
+    <p>Star rating select uses radio buttons under the hood so even in the absence of CSS, its gracefully degradation should allow for full accessible interactivity and use.</p>
+
+    <p><strong>NOTE:</strong> JavaScript is required to handle the filled visual marking of all stars previous to the currently selected star rating.</p>
+
+    <p>
+        <div class="star-rating-select">
+            <span class="radio">
+                <input aria-label="1 Star" class="radio__control" type="radio" name="radio-star-rating-select" value="1">
+                <span class="radio__icon">
+                    <svg class="star-rating-select__icon" focusable="false">
+                        {% include symbol.html name="star-dynamic" %}
+                    </svg>
+                </span>
+            </span>
+
+            <span class="radio">
+                <input aria-label="2 Stars" class="radio__control" type="radio" name="radio-star-rating-select" value="2">
+                <span class="radio__icon">
+                    <svg class="star-rating-select__icon" focusable="false">
+                        {% include symbol.html name="star-dynamic" %}
+                    </svg>
+                </span>
+            </span>
+
+            <span class="radio">
+                <input aria-label="2 Stars" class="radio__control" type="radio" name="radio-star-rating-select" value="3">
```

Comment:
Just double check these aria-label values (should be "3 Stars" here), same thing in stories file.